doc/src/neighbor.rst

```diff
@@ -55,14 +55,24 @@ For the *bin* style, the bin size is set to 1/2 of
 the largest cutoff distance between any pair of atom types and a
 single set of bins is defined to search over for all atom types.  This
 can be inefficient if one pair of types has a very long cutoff, but
-other type pairs have a much shorter cutoff.  For style *multi* the
-bin size is set to 1/2 of the shortest cutoff distance and multiple
-sets of bins are defined to search over for different atom types.
+other type pairs have a much shorter cutoff. The *multi* style uses
+different sized bins for collections of different sized particles. Different
+sets of bins are then used to construct the neighbor lists as as further 
+described by Shire, Hanley, and Stratford :ref:`(Shire) <bytype-Shire>`. 
 This imposes some extra setup overhead, but the searches themselves
-may be much faster for the short-cutoff cases.
-See the :doc:`comm_modify mode multi <comm_modify>` command for a
-communication option that may also be beneficial for simulations of
-this kind.
+may be much faster. By default, separate collections of particles are defined 
```

Comment:
By default, each atom type defines a separate collection of particles.  For systems where two or more atom types have the same size (either physical size or cutoff distance), the definition of collections can be customized, which can result in less overhead and faster performance.  See the neigh_modify XYZ command for how to define custom collections.  Whether the collection definition is customized or not, also see the comm_modify mode bytype command for communication options that further improve performance in a manner consistent with neighbor style multi.